build_tools/upload_logs_to_s3.py

```diff
@@ -47,26 +54,32 @@ def upload_logs_to_s3(s3_base_path: str, build_dir: Path):
         log(f"[INFO] Log directory {log_dir} not found. Skipping upload.")
         return
 
+    if not s3_base_path.startswith("s3://"):
+        log(f"[ERROR] Invalid s3_base_path: {s3_base_path}")
+        sys.exit(2)
+
+    bucket, *prefix_parts = s3_base_path[5:].split("/", 1)
```

Comment:
I think this might be error prone depending on the endpoint. Maybe go closer with what I've done in `generate_release_index.py`:
https://github.com/ROCm/TheRock/blob/b354440c1e585d2aeca63209f2b9b4b5dac5e3a3/build_tools/packaging/python/generate_release_index.py#L30-L45

Reply:
Updated the code.